adagucserverEC/LayerTypeLiveUpdate/LayerTypeLiveUpdate.cpp

```diff
@@ -90,8 +193,13 @@ int layerTypeLiveUpdateRenderIntoImageDataWriter(CDataSource *dataSource, CServe
   }
 
   std::vector<CDataSource *> dataSourceRef = {dataSource->clone()};
-  status = imageDataWriter.addData(dataSourceRef);
-  CDBDebug("Adding data status was %d", status);
+
+  if (srvParam->requestType == REQUEST_WMS_GETFEATUREINFO) {
+    status = imageDataWriter.getFeatureInfo(dataSourceRef, 0, int(srvParam->dX), int(srvParam->dY));
```

Comment:
You could calculate the pixel only for srvParam->dX and srvParam->dY in the datapostproc.